### PR TITLE
Finishing

### PR DESCRIPTION
**Pull request by feleck**
Updates of models - added users, controllers and views users and admins, polishing styles.

### CODE REVIEW

File: app/controllers/comments_controller.rb [hunk 1]
```diff
@@ -1,13 +1,17 @@
 class CommentsController < ApplicationController
+  before_action :authenticate_user!
+
   def create
     @post = Post.find(params[:post_id])
-    @comment = @post.comments.create(comment_params)
+    @comment = @post.comments.new(comment_params)
+    @comment.user = current_user
+    @comment.save
```
Comment (oskarlakner) [inline app/controllers/comments_controller.rb]: how about error handling? ;)

File: app/controllers/posts_controller.rb [hunk 2]
```diff
@@ -1,5 +1,5 @@
 class PostsController < ApplicationController
-  before_action :authenticate_user!, only: %w[edit update destroy]
+  before_action :authenticate_user!, only: %w[create edit update destroy]
```
Comment (oskarlakner) [inline app/controllers/posts_controller.rb]: in this case imo it's better to use `except`

File: app/controllers/posts_controller.rb [hunk 3]
```diff
@@ -15,6 +15,7 @@ def new
 
   def create
     @post = Post.new(post_params)
+    @post.user = current_user
```
Comment (oskarlakner) [inline app/controllers/posts_controller.rb]: I like to merge user to permitted params

File: db/schema.rb [hunk 4]
```diff
@@ -10,14 +10,14 @@
...
-ActiveRecord::Schema.define(version: 2018_07_13_083524) do
+ActiveRecord::Schema.define(version: 2018_07_17_095526) do
 
   create_table "comments", force: :cascade do |t|
-    t.string "author"
     t.text "body"
     t.integer "post_id"
     t.datetime "created_at", null: false
     t.datetime "updated_at", null: false
+    t.integer "user_id"
```
Comment (oskarlakner) [inline db/schema.rb:20]: how about index?

**Review by oskarlakner — CHANGES_REQUESTED** [review]

File: db/schema.rb (hunk 4 above)
Comment (feleck) [reply db/schema.rb:20]: index added to user_id in comments table

File: app/controllers/comments_controller.rb [hunk 5]
```diff
@@ -1,13 +1,19 @@
...
+    else
+      redirect_to @post, flash: { error: "Can't add empty comment!" }
+    end
   end
 
   private
 
   def comment_params
-    params.require(:comment).permit(:author, :body)
+    params.require(:comment).permit(:body).merge(user_id: current_user.id)
```
Comment (oskarlakner) [inline app/controllers/comments_controller.rb]: it's fine, but in rails you can merge an object aswell

File: app/views/comments/_comment.haml [hunk 6]
```diff
@@ -0,0 +1,7 @@
+.comment
+  - @post.comments.each do |comment|
```
Comment (oskarlakner) [inline app/views/comments/_comment.haml]: is it n+1? ;)

**Review by oskarlakner — APPROVED** [review]

**Pull request merged**